### ISSUE TITLE
Cloudflare API Token does not work

### ISSUE THREAD

**Issue by belaytzev** [go-acme/lego#968]
I try use _CLOUDFLARE_API_TOKEN_ and _CLOUDFLARE_API_TOKEN_FILE_ but lego require _CLOUDFLARE_EMAIL_ ,_CLOUDFLARE_API_KEY_ every time

```
# CLOUDFLARE_API_TOKEN=XXXXXXXXXXXXXXXXXXXXXXXXXXXXX lego --dns cloudflare --domains example.com --email email@example.com run
cloudflare: some credentials information are missing: CLOUDFLARE_EMAIL,CLOUDFLARE_API_KEY
```

reproduced on versions:
* 3.0.2
* 3.0.0
* 2.7.2

**Comment by ldez** [comment]
`CLOUDFLARE_API_TOKEN` will be available in v3.1

**Comment by belaytzev** [comment]
@ldez Thanks, then I will be waiting v3.1
